README.yaml

```diff
@@ -126,6 +127,60 @@ usage: |-
     ]
   }
   ```
+
+  With [enhanced monitoring](examples/enhanced_monitoring)
+
+  ```hcl
+  # create IAM role for monitoring
+  resource "aws_iam_role" "iam_role" {
```

Comment:
rename  `iam_role` to `enhanced_monitoring` or just `default` (if there is only one `aws_iam_role` in the module).

Reply:
ahh, yeah, that was already done in the examples file, just need to copy that over to the README.yaml and rebuild.